lib/src/enums.dart

```diff
@@ -0,0 +1,28 @@
+/// Values for the event name to be sent to Google Analytics
+///
+/// The [label] for each enum value is what will be logged, the [desc]
+/// is here for documentation purposes
+enum DashEvents {
+  hotReloadTime(
+    label: 'hot_reload_time',
+    desc: 'Hot reload duration',
+  ),
+  ;
+
+  final String label;
+  final String desc;
+  const DashEvents({
+    required this.label,
+    required this.desc,
+  });
+}
+
+/// Enumerate options for platform
+enum DevicePlatforms {
```

Comment:
```suggestion
enum DevicePlatform {
```

Reply:
Where and for what purpose is this used? Are we not currently sending more detailed operating system information with each event?

Reply:
This is being used by the tools implementing this package. The idea was to have each client tool determine what their OS is, then match it against one of these enum values. Once they have assigned an enum value, we will have all relevant information around the description, tool owner, and anything else we want to associate with OS's

It is better than having each client send a string representation of the OS which could vary (ie. macos, macOS, etc.)

Reply:
Two things:

1. Why not have this package detect the operating system?
2. Ahh, I thought that we sent the OS version with each event, but I was wrong.

Reply:
Good idea, I've moved the logic into the package now